content/efcore/getting-started/what-is-ef-core.md

```diff
@@ -0,0 +1,5 @@
+# Intro to EF Core
+
+Entity Framework Core (EF Core) is a lightweight, extensible, and cross-platform version of Entity Framework. Entity Framework is an object-relational mapper (O/RM) that enables .NET developers to work with a database using .NET objects. This allows developers to create and manage databases without most of the data-access code that developers usually need to write. 
+
+EF Core does not have all the features available in the latest version of Entity Framework (EF6.x). However, EF Core does offer a few features EF6.x does not. For more on these differences, check out [the docs](https://docs.efproject.net/en/latest/efcore-vs-ef6/features.html). If you have further questions about which version is best for your project, visit ["Which One Is Right for You?"](https://docs.efproject.net/en/latest/efcore-vs-ef6/choosing.html).
```

Comment:
### Suggested Edit:

EF Core and the latest version of Entity Framework (EF6.x) differ in the features they support. A comparison of the features supported by each can be found on the [features page](https://docs.efproject.net/en/latest/efcore-vs-ef6/features.html) of the docs. If you want help determining which version is best for your project, check out [this page](https://docs.efproject.net/en/latest/efcore-vs-ef6/choosing.html) of the docs.